provisioner/auth/accesskeys.go

```diff
@@ -0,0 +1,9 @@
+package auth
```

Comment:
why not v1alpha1 package?

Reply:
That's a good question.  This isn't an API object but it is a requirement for creating API objects.  I believe the k8s convention is to co-locate some of these helper structs with the CRD apis.